checkout/checkout_service.bal

```diff
@@ -75,7 +76,11 @@ service "CheckoutService" on new grpc:Listener(9094) {
         self.emailClient = check new ("http://" + emailHost + ":9097");
     }
 
-    remote function PlaceOrder(PlaceOrderRequest request) returns PlaceOrderResponse|error {
+    # Places the order and process payment, shipping and email notification.
+    #
+    # + request - `PlaceOrderRequest` containing user details
+    # + return - returns `PlaceOrderResponse` containing order details
+    isolated remote function PlaceOrder(PlaceOrderRequest request) returns PlaceOrderResponse|error {
```

Comment:
Should we explicitly mark these functions isolated? AFAIK remember in a code review, someone said its not a must. 
What is the correct practice here? @MaryamZi

Reply:
Since this is a service declaration, and `isolated` is inferred, we don't have to explicitly mark the service object and/or the methods as `isolated`. Ideally, `isolated` should be inferred here. @dilanSachi, can you check by removing `isolated`?

But if this was a public service class or a class exposed via some public construct, `isolated` won't be inferred. In such cases we need to explicitly mark as `isolated`.

Reply:
Fixed in https://github.com/ballerina-guides/gcp-microservices-demo/pull/27